### PR TITLE
Tidy cloudformation.

### PR DESCRIPTION
**Pull request by LATaylor-guardian**

### CODE REVIEW

File: cloudformation.json [hunk 1]
```diff
@@ -190,17 +193,18 @@
...
-                { "Fn::Join": [ "", [ "wget https://s3-eu-west-1.amazonaws.com/content-api-dist/content-api/",{ "Ref": "Stage" }, "/bonobo/bonobo-1.0-SNAPSHOT.tgz" ] ] },
-                { "Fn::Join": [ "", [ "wget https://s3-eu-west-1.amazonaws.com/content-api-dist/content-api/",{ "Ref": "Stage" }, "/bonobo/bonobo.service" ] ] },
+                { "Fn::Join": [ "", [ "aws --region eu-west-1 s3 cp s3://content-api-dist/content-api/",{ "Ref": "Stage" }, "/bonobo/bonobo-1.0-SNAPSHOT.tgz ." ] ] },
+                { "Fn::Join": [ "", [ "aws --region eu-west-1 s3 cp s3://content-api-dist/content-api/",{ "Ref": "Stage" }, "/bonobo/bonobo.service /etc/systemd/system/bonobo.service" ] ] },
+
                 "tar -xvf bonobo-1.0-SNAPSHOT.tgz",
                 "mv bonobo-1.0-SNAPSHOT bonobo",
-                "mv bonobo.service /etc/systemd/system",
-                { "Fn::Join": [ "", [ "aws s3 cp s3://content-api-config/bonobo/",{ "Ref": "Stage" }, "/bonobo.conf /etc/gu/bonobo.conf" ] ] },
+                { "Fn::Join": [ "", [ "aws --region eu-west-1 s3 cp s3://content-api-config/bonobo/",{ "Ref": "Stage" }, "/bonobo.conf /etc/gu/bonobo.conf" ] ] },
```
Comment (mchv) [inline cloudformation.json]: would be great to have the region as a parameter and configure it globally like `aws set region eu-west-1`

Comment (LATaylor-guardian) [inline cloudformation.json]: Yeah, I did think about this but was lazy! Will add it.

**Comment by LATaylor-guardian** [comment]
Tested on CODE.

**Comment by LATaylor-guardian** [comment]
Keep cloudformation as is (without ANY modification) when updating the PROD stack.

**Comment by cb372** [comment]
:+1: but :fearful: 

Let's take a backup of Dynamo first!

**Comment by LATaylor-guardian** [comment]
Closing. Given the current state of the cloudformation, this is not possible to apply.

**Pull request closed without merging**